engines/dradis-api/lib/dradis/ce/api/basic_auth_strategy.rb

```diff
@@ -17,11 +9,10 @@ def valid?
 
     def authenticate!
       if auth.provided? && auth.basic? && auth.credentials
```

Comment:
even if this is how the code _was_ these checks belong to `#valid?` and not to `#authenticate!`